modin/pandas/test/test_io.py

```diff
@@ -606,6 +606,40 @@ def test_read_csv_mangle_dupe_cols(self):
         str_non_unique_cols = "col,col,col,col\n" "5, 6, 7, 8\n" "9, 10, 11, 12\n"
         eval_io_from_str(str_non_unique_cols, unique_filename, mangle_dupe_cols=True)
 
+    # NA and Missing Data Handling tests
+    @pytest.mark.parametrize("na_values", ["custom_nan", "73"])
+    @pytest.mark.parametrize("keep_default_na", [True, False])
+    @pytest.mark.parametrize("na_filter", [True, False])
+    @pytest.mark.parametrize("verbose", [True, False])
+    @pytest.mark.parametrize("skip_blank_lines", [True, False])
+    def test_read_csv_na_handling(
+        self,
+        make_csv_file,
+        na_values,
+        keep_default_na,
+        na_filter,
+        verbose,
+        skip_blank_lines,
+    ):
+        kwargs = {
+            "na_values": na_values,
+            "keep_default_na": keep_default_na,
+            "na_filter": na_filter,
+            "verbose": verbose,
+            "skip_blank_lines": skip_blank_lines,
+        }
+        unique_name = get_unique_filename("test_read_csv_na_handling", kwargs)
```

Comment:
Getting a unique name is different than how it's done in #2331. Why?

Reply:
This is done because https://github.com/modin-project/modin/pull/2331 proposes new mechanism of `unique_name` creation, which doesn`t need any parameters. This PR doesn't have this mechanism, so parameters are needed.

Reply:
Please update this place to your new mechanism using class fixture.

Reply:
@anmyachev fixture is added.